client/app/cloud/project/billing/cloud-project-billing-legacy.service.js

```diff
@@ -0,0 +1,297 @@
+angular.module('managerApp')
+  .service('CloudProjectBillingLegacyService', function CloudProjectBillingLegacyService($q, OvhApiMe) {
+    const self = this;
+
+    function roundNumber(number, decimals) {
+      return Number((`${Math.round(`${number}e${decimals}`)}e-${decimals}`));
```

Comment:
`e` ?

I don't understand what this does ?

Also, don't use `Number`, it is a primitive type constructor, we should never use primitive type constructors :)